src/utils/validation.js

```diff
@@ -0,0 +1,24 @@
+export const Validation = input => {
```

Comment:
```suggestion
export const userInputValidation = input => {
```
유저 인풋을 유효성 검사하는 로직만 있는 것 같아서 함수 네이밍으로 `userInputValidation` 유저 인풋을 유효성 검사한다는 것을 알려줘도 좋을 것 같아요! 

기존 `Validation` 함수 이름은 유저 인풋이 아니여도 범용적으로 사용될 수 있는 로직이라고 오해할 수도 있을 것 같아요🙃

Reply:
맞습니다! userInput만 검사를 진행하기 때문에 오해가 있을수있겠네요!